src/main/typescript/components/Header.tsx

```diff
@@ -88,24 +88,17 @@ const Header = () => {
             dispatch(getUser())
     })
 
-    const loginNavBar = isLoggedIn && loginName
-        ? [
-            <span key="loginName" className="navbar-text">{loginName}</span>,
-            <NavBarLink key="my datasets" to={depositOverviewRoute}>My Datasets</NavBarLink>,
-            <NavBarLink className="logoff"
-                        key="log out"
-                        to={signoutRoute}
-                        title="Log out">Log out</NavBarLink>,
-        ]
-        : [
-            <NavBarLink key="login" to={loginRoute} title="Login to EASY">Log in</NavBarLink>,
-        ]
+    const logoutNavBar = [
```

Comment:
now that the ternary operator isn't here anymore, you can just inline this `logoutNavBar` into line 101.